docs/OCP_CI_Tutorials/Scenarios/Orchestration_Guide.md

```diff
@@ -0,0 +1,81 @@
+# OpenShift CI Scenario Test Orchestration Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Introduction](#introduction)
+- [Scenario Orchestration](#scenario-orchestration)
+  - [Using a Chain](#using-a-chain)
+  - [Using a Ref](#using-a-ref)
+
+## Introduction
+This document aims to define and guide you through creating the "Orchestration" step of a scenario.
+
+We define the Orchestration step of a scenario as:
+
+> Completing any setup or configuration necessary to execute tests. This setup can include things like: installing an operator, retrieving any necessary variables, additional configuration of the test cluster, deployment of any necessary resources on the test cluster, etc.
+
+Please follow the guide below to create the orchestration step of your scenario.
+
+## Scenario Orchestration
+There are a couple of ways to go about creating the orchestration step of a scenario, depending on the amount of work that need to be done during this step. If there isn't much work to be done and it can all be done in a single container using a single BASH script, a ref can be used for this step. If there is a lot of work to be done and you need to utilize steps that aren't specific to this scenario, you'll need to use a chain (most common).
```

Comment:
```suggestion
There are a couple of ways to go about creating the orchestration step of a scenario, depending on the amount of work that needs to be done during this step. If there isn't much work to be done and it can all be done in a single container using a single BASH script, a ref can be used for this step. If there is a lot of work to be done and you need to utilize steps that aren't specific to this scenario, you'll need to use a chain (most common).
```